tests/unit-tests/tests-email-tags.php

```diff
@@ -34,7 +34,14 @@ function test_give_email_tag_first_name() {
 		$this->assertEquals( 'Admin', $firstname );
 
 		/*
-		 * Case 3: First name with filter
+		 * Case 3: First name from donor_id.
+		 */
+		$donor = Give_Helper_Donor::create_simple_payment();
```

Comment:
@raftaar1191 Method name should be `create_donor` instead of `create_simple_payment` and implement is wherever applicable